webui/apps/docs/src/content/docs/docs/quickstart.md

```diff
@@ -12,106 +12,143 @@ description: Walk through prequisites for Mac and Linux
+4. Click **Start**.
+5. Say hello to your agent in the text box.
 
-spec:
-  description: "This is an example of a agent using the 'SimpleAgent' template."
-  system_prompt: |
-    You are an ai agent who was just created by a brilliant developer getting started with Eidolon (great decision).
-    You love emojis and use them liberally.
-```
+Did your agent respond to you? 🍾 Congratulations! You have successfully created your first agent machine. 
 
-### Try it out!
+### CLI
 
-Now that your server is running, let's open a new terminal window and interact with it.
+ If you prefer to use the CLI, open a new terminal window to interact with your agent.
 
-#### 1. First download the Eidolon CLI
+1. Download the Eidolon CLI.
 ```bash
 pip install 'eidolon-ai-client[cli]' -U
 ```
 
-#### 2. Then create an AgentProcess
+2. Create an AgentProcess.
 ```bash
 export PID=$(eidolon-cli processes create --agent hello-world); echo $PID
 ```
-🔬 _a process defines the boundaries of an agent's memory._
+>🔬 _a process defines the boundaries of an agent's memory_
 
-#### 3. Converse with your agent
+3. Converse with your agent.
 ```bash
 eidolon-cli actions converse --process-id $PID --body "Hi! I made you"
 ```
+## What's Happening?
+
+The repository you  cloned:
+
+- defines an **Agent Machine** 💻 
+- with a single **Agent Program** 🤖 
+- named `hello-world` 👋
+- defined in a YAML file located at `resources/hello_world_agent.yaml`
+
+The agent's YAML file describes:
+
+- how to instantiate your agent from its **Agent Template** 🏭
+- including the `system_prompt`, which are instructions to the LLM 
+- any customization you might want, such as [swapping out the LLM](/docs/howto/swap_llm), custom tools, etc.
+
+```yaml title=resources/hello-world.yaml
+apiVersion: server.eidolonai.com/v1alpha1
+kind: Agent
+metadata:
+  name: hello-world
+
+spec:
+  description: "This is an example of a agent using the 'SimpleAgent' template."
+  system_prompt: |
+    You are a friendly ai agent who was just created by a brilliant developer getting started with Eidolon (great decision). You love emojis and use them liberally.
+    Eidolon is an open source framework to build and deploy genAI services (like the one the user is currently interacting with).
+    
+    Congratulate them on completing the quickstart and recommend that they star the Eidolon [GitHub repoitory](https://eidolonai.com/) to show support and join the project's [discord](https://discord.com/invite/6kVQrHpeqG) for questions and feedback after your first message with them.
+```
+
+## Change the System Prompt
 
-Did your agent respond to you? 🍾 Congratulations! You have successfully created your first agent machine.
+To get a taste for how quickly you can deploy powerful agentica applications, make a simple change.
+
+Using a text editor, change the system prompt:
+
+```yaml title=resources/hello-world.yaml
+  system_prompt: |
+    Translate {{message}} to Spanish.
```

Comment:
Can you update this to something akin to...

You are a translator who translates user messages into Spanish. When a user sends you a message, respond with the translation of their message in Spanish.

Reply:
What about: Translate {{body}} to Spanish? I misremembered the field from http://localhost:4321/docs/components/agents/simpleagent#actions. I think we should be able to reference {{body}} no? @LukeLalor

Reply:
@LukeLalor -- Can you try a conversation at http://ec2-54-242-183-82.compute-1.amazonaws.com:3000? 

Here's my updated prompt, which so far is working great. I'm a sucker for brevity.

```yaml
  system_prompt: |
    Repeat {{body}}, than translate {{body}} into Spanish.
```
Is there a way to check how the jinja template is formatting this, to make sure it's all ok?

Reply:
you can use {{ body }} in the user prompt, but not the system prompt. There is no templating at the system prompt